addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -299,7 +299,7 @@ registerModel({
         attachmentLists: many2many('mail.attachment_list', {
             inverse: 'attachments',
         }),
-        attachmentViewer: many2many('mail.attachment_viewer', {
+        attachmentViewer: many2many('AttachmentViewer', {
```

Comment:
for later but attachmentViewer here should be plural as it's x2many

Reply:
see https://github.com/odoo/odoo/pull/81467